example/server/src/index.ts

```diff
@@ -369,14 +369,17 @@ app.post('/create-setup-intent', async (req, res) => {
 
   //@ts-ignore
   const setupIntent = await stripe.setupIntents.create({
-    ...{ customer: customer.id, payment_method_types },
```

Comment:
I would not change the index.ts ourselves because we copy it from stripe. I will do an update here. Can you revert this?